src/functionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+
+const TodoContainer = () => {
+  function getInitialTodos() {
+    // getting stored items
+    const temp = localStorage.getItem('todos');
+    const savedTodos = JSON.parse(temp);
+    return savedTodos || [];
+  }
```

Comment:
[OPTIONAL] This function is similar to `useEffect` at the state makes it act like  `DidComponentMount()` for that reason consider using it which makes you use less effort and helps your team member to easily understand it (despite it being super clear) but just as an example

Reply:
I appreciate a lot what you are saying with this comment, but I think to use this function for future reference.